modules/elasticsearch_connector_views/src/Plugin/views/query/ElasticsearchViewsQuery.php

```diff
@@ -300,25 +323,6 @@ public function build(ViewExecutable $view) {
     $this->params['fields'][] = '_source';
 
     $params = array();
-    if (isset($this->params['q']) && !empty($this->params['q'])) {
```

Comment:
Perhaps this is out of scope? Can you elaborate on what's being removed here?

Reply:
This must have been the result of merging the base branch on top.